docs/sources/installation/debian.md

```diff
@@ -22,12 +26,12 @@ If you install from the APT repository, then Grafana is automatically updated ev
 
 | Grafana Version           | Package            | Repository                                                |
 | ------------------------- | ------------------ | --------------------------------------------------------- |
-| Grafana OSS               | grafana            | `https://packages.grafana.com/oss/deb stable main`        |
-| Grafana OSS (Beta)        | grafana            | `https://packages.grafana.com/oss/deb beta main`          |
 | Grafana Enterprise        | grafana-enterprise | `https://packages.grafana.com/enterprise/deb stable main` |
 | Grafana Enterprise (Beta) | grafana-enterprise | `https://packages.grafana.com/enterprise/deb beta main`   |
+| Grafana OSS               | grafana            | `https://packages.grafana.com/oss/deb stable main`        |
+| Grafana OSS (Beta)        | grafana            | `https://packages.grafana.com/oss/deb beta main`          |
 
-> We recommend all users install the Enterprise Edition of Grafana, which can be seamlessly upgraded with a Grafana Enterprise [subscription](https://grafana.com/products/enterprise/?utm_source=grafana-install-page).
+> The Enterprise Edition is the default and recommended edition. It includes all the features of the OSS Edition, can be used for free and can be upgraded to the [full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page), including support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
```

Comment:
```suggestion
>**Note:** The Enterprise is the recommended and default option. It is available for free and includes all the features of the OSS Edition. You can also upgrade to the [ full Enterprise feature set](https://grafana.com/products/enterprise/?utm_source=grafana-install-page) and has support for [Enterprise plugins](https://grafana.com/grafana/plugins/?enterprise=1&utcm_source=grafana-install-page).
```

Reply:
In the table above, I see only one Enterprise offering. But from the note, it seems like there are two enterprise offerings, one free and one licensed. Do we need to update the table? Thank you.

Reply:
Should this be free, pro, and advanced Cloud tiers instead?